### ISSUE TITLE
Issue: Null pointer exception

### ISSUE THREAD

**Issue by acurat** [avishek-sen-gupta/cobol-rekt#62]
This below code fails to Null pointer

Exception stack trace:
```
[2024-11-09 00:06:53] [FINE] Building flow for EVALUATEBILLMSTR-CODE-STAT-DSTRWHEN(' ')CONTINUEWHEN('P')MOVE'Y'TOFG-ERROR-FLAGMOVE'ORDER HAS BEEN POSTED - VIEW THRU DSPR 'TOerrorMessageMOVE'9101'TOerrorCodeWHENOTHERMOVE'Y'TOFG-ERROR-FLAGMOVE'ORDER HAS BEEN RELEASED TO BILL 'TOerrorMessageMOVE'9102'TOerrorCodeEND-EVALUATE 
[2024-11-09 00:06:53] [FINE] Building flow for EVALUATE 
java.lang.NullPointerException: Cannot invoke "org.eclipse.lsp.cobol.core.CobolParser$EvaluateThroughContext.evaluateValue()" because the return value of "org.eclipse.lsp.cobol.core.CobolParser$EvaluateConditionContext.evaluateThrough()" is null
	at org.smojol.common.vm.expression.EvaluateBreaker.condition(EvaluateBreaker.java:83)
	at org.smojol.common.vm.expression.EvaluateBreaker.mustHaveConditions(EvaluateBreaker.java:64)
	at org.smojol.common.vm.expression.EvaluateBreaker.recursiveOr(EvaluateBreaker.java:58)
	at org.smojol.common.vm.expression.EvaluateBreaker.conditionGroup(EvaluateBreaker.java:52)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1708)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575)
	at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260)
	at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616)
	at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:622)
	at java.base/java.util.stream.ReferencePipeline.toList(ReferencePipeline.java:627)
	at org.smojol.common.vm.expression.EvaluateBreaker.decompose(EvaluateBreaker.java:41)
	at org.smojol.toolkit.ast.EvaluateFlowNode.buildInternalFlow(EvaluateFlowNode.java:42)
	at org.smojol.toolkit.ast.CobolFlowNode.buildFlow(CobolFlowNode.java:45)
	at org.smojol.toolkit.ast.ConditionalStatementFlowNode.buildInternalFlow(ConditionalStatementFlowNode.java:33)
	at org.smojol.toolkit.ast.CobolFlowNode.buildFlow(CobolFlowNode.java:45)
	at org.smojol.toolkit.ast.CompositeCobolFlowNode.buildInternalFlow(CompositeCobolFlowNode.java:51)
	at org.smojol.toolkit.ast.CobolFlowNode.buildFlow(CobolFlowNode.java:45)
	at org.smojol.toolkit.ast.IfFlowNode.buildInternalFlow(IfFlowNode.java:41)
_omitted for brevity_ 
```

Code:

```
       2000-PROCESS-REQUEST.
      *****************************************************************

           PERFORM 9000-SELECT-MAX-BILL-SUPP
           IF FG-ERROR-FLAG = 'N'
              MOVE BILLMSTR-NBR-BILL-SUPP TO WS-NBR-BILL-SUPP
              PERFORM 9100-READ-BILLMSTR

              IF FG-ERROR-FLAG = 'N'
                 EVALUATE BILLMSTR-CODE-STAT-DSTR
                    WHEN (' ')
                       CONTINUE

                    WHEN ('P')
                       MOVE 'Y' TO FG-ERROR-FLAG
                       MOVE 'ORDER HAS BEEN POSTED - VIEW THRU DSPR '
                                            TO errorMessage
                       MOVE '9101' TO errorCode

                    WHEN OTHER
                       MOVE 'Y' TO FG-ERROR-FLAG
                       MOVE 'ORDER HAS BEEN RELEASED TO BILL '
                                            TO errorMessage
                       MOVE '9102' TO errorCode
                 END-EVALUATE
              END-IF
           END-IF
```

**Comment by avishek-sen-gupta** [comment]
This should be fixed in commit [00a0f8bb495ce0dee67c5260a42a9fb22cbc0963](https://github.com/avishek-sen-gupta/cobol-rekt/commit/00a0f8bb495ce0dee67c5260a42a9fb22cbc0963). Could you check?

**Comment by acurat** [comment]
This works. Thanks. Closing issue.
